pkg/resourcemanager/mysql/server/reconcile.go

```diff
@@ -190,9 +190,9 @@ func (m *MySQLServerClient) Ensure(ctx context.Context, obj runtime.Object, opts
 			instance.Status.Provisioned = false
 			instance.Status.FailedProvisioning = true
 			return true, nil
-
 		}
 
+		instance.Status.PollingURL = pollURL
```

Comment:
Should we clear this when the operation is done?

Reply:
Yeah, good idea - I'll clear it when we set Provisioned to true. It's probably still useful until then.